src/pybind/mgr/dashboard/module.py

```diff
@@ -977,18 +976,18 @@ def _osds_by_server(self):
                             summary = self._osd_summary(osd_id,
                                                         osd_map.osds_by_id[osd_id])
 
-                            if first:
-                                # A little helper for rendering
-                                summary['first'] = True
-                                first = False
                             result[hostname].append(summary)
 
+                    result[hostname].sort(key=lambda a: a['id'])
```

Comment:
what's the type of `a['id']`? we should use `sort(key=lambda a: int(a'id'])` if it's `str`.

Reply:
please see `PyModules::dump_server()`.

Reply:
yeah. it's correct